[2025-10-01T01:55:10.993+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-01T01:55:11.001+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_forecast_stock_robust.build_union_table scheduled__2025-09-30T00:00:00+00:00 [queued]>
[2025-10-01T01:55:11.003+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_forecast_stock_robust.build_union_table scheduled__2025-09-30T00:00:00+00:00 [queued]>
[2025-10-01T01:55:11.004+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-10-01T01:55:11.007+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): build_union_table> on 2025-09-30 00:00:00+00:00
[2025-10-01T01:55:11.010+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=144) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-01T01:55:11.011+0000] {standard_task_runner.py:64} INFO - Started process 146 to run task
[2025-10-01T01:55:11.011+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ml_forecast_stock_robust', 'build_union_table', 'scheduled__2025-09-30T00:00:00+00:00', '--job-id', '453', '--raw', '--subdir', 'DAGS_FOLDER/ml_forecast_stock.py', '--cfg-path', '/tmp/tmpqj6gytcc']
[2025-10-01T01:55:11.012+0000] {standard_task_runner.py:91} INFO - Job 453: Subtask build_union_table
[2025-10-01T01:55:11.029+0000] {task_command.py:426} INFO - Running <TaskInstance: ml_forecast_stock_robust.build_union_table scheduled__2025-09-30T00:00:00+00:00 [running]> on host 11e1994bdecf
[2025-10-01T01:55:11.065+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data226' AIRFLOW_CTX_DAG_ID='ml_forecast_stock_robust' AIRFLOW_CTX_TASK_ID='build_union_table' AIRFLOW_CTX_EXECUTION_DATE='2025-09-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-30T00:00:00+00:00'
[2025-10-01T01:55:11.066+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-01T01:55:11.108+0000] {base.py:84} INFO - Using connection ID 'snowflake_default' for task execution.
[2025-10-01T01:55:11.109+0000] {connection.py:408} INFO - Snowflake Connector for Python Version: 3.11.0, Python Version: 3.12.4, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-01T01:55:11.110+0000] {connection.py:1258} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-01T01:55:12.130+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-01T01:55:12.302+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-01T01:55:12.398+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-01T01:55:12.399+0000] {ml_forecast_stock.py:223} INFO - Using: warehouse="BEETLE_QUERY_WH", database="ASH_DB", schema="STOCK_SCHEMA"
[2025-10-01T01:55:12.519+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-01T01:55:12.520+0000] {ml_forecast_stock.py:231} INFO - Executing SQL statement 1/2
[2025-10-01T01:55:13.770+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-01T01:55:13.770+0000] {ml_forecast_stock.py:231} INFO - Executing SQL statement 2/2
[2025-10-01T01:55:14.590+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-01T01:55:14.706+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-01T01:55:14.707+0000] {ml_forecast_stock.py:237} INFO - All SQL statements executed successfully
[2025-10-01T01:55:14.707+0000] {connection.py:773} INFO - closed
[2025-10-01T01:55:14.776+0000] {connection.py:779} INFO - No async queries seem to be running, deleting session
[2025-10-01T01:55:14.845+0000] {ml_forecast_stock.py:398} INFO - Union table built successfully
[2025-10-01T01:55:14.847+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-01T01:55:14.848+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-01T01:55:14.862+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ml_forecast_stock_robust, task_id=build_union_table, run_id=scheduled__2025-09-30T00:00:00+00:00, execution_date=20250930T000000, start_date=20251001T015511, end_date=20251001T015514
[2025-10-01T01:55:14.914+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-01T01:55:14.928+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-01T01:55:14.930+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
